src/help/zaphelp/contents/ui/tabs/spider.html

```diff
@@ -48,8 +48,26 @@ <h1>Spider tab</h1>
 	</ul>
 
 	<p>For each spider message, shown under the Messages tab, you can see details of the request sent and response
-		received, moreover it's indicated, in the column <code>Processed</code>, whether or not the message was processed
-		(that is, the response was successfully received and parsed).
+		received, moreover it's indicated, in the column <code>Processed</code>, whether or not the message was processed:
+	<ul>
+		<li><img align="bottom" alt="pause button" src="../../images/16/152.png">Successfully
```

Comment:
I think just `Successful -` vs "Successfully -"

Reply:
That's the text shown for that state (I can change in the UI though).

Reply:
No it's ok, for the context I guess it's fine. I wasn't considering the column name, I just read it as a sentence...